pkg/server/server.go

```diff
@@ -180,42 +183,42 @@ func NewServer(o *Options) (*Server, error) {
 		).ClientConfig()
 	}
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("server creation failed for kubeconfig [%s] master[%s]: %w", o.Kubeconfig, o.master, err)
```

Comment:
I prefer 'master URL' to 'master' in error message as above `klog.Info("Neither kubeconfig file nor master URL was specified. Falling back to in-cluster config.")`. Could you please change it?